guides/source/active_record_querying.md

```diff
@@ -1266,6 +1266,9 @@ class Client < ActiveRecord::Base
 end
 ```
 
+NOTE: The `default_scope` is also applied while creating/building a record.
+It is not applied while updating a record.
```

Comment:
This one have an example.
`NOTE: Using where like this will only work when you pass it a Hash. For SQL-fragments you need to use references to force joined tables:`